pages/api/search/issues/total.ts

```diff
@@ -38,6 +38,17 @@ async function getTotal(req: NextApiRequest, res: NextApiResponse) {
     if (!network) return res.status(404).json("Invalid network");
 
     whereCondition.network_id = network?.id;
+  } else {
+    const networks = await models.network.findAll({
+      where: {
+        isRegistered: true,
+        isClosed: false
+      }
+    })
+
+    if (networks.length === 0) return res.status(404).json("Networks not found");
```

Comment:
answer with a json message instead or switch to `text()`

Reply:
Is the standard we are following on endpoints

![image](https://user-images.githubusercontent.com/34746557/212089292-cf42af30-2ad5-41dd-88fc-fe6fc4e1440b.png)